### DISCUSSION TITLE
git commit -v results in plugin needing to update

### DISCUSSION THREAD

**Discussion by iton0** [nvim-tree/nvim-tree.lua#2674] [Q&A]
Whenever I use git commit -v and load into the commit message my nvim tree says it needs to be updated saying version 0.100.0 is available:

![Screenshot from 2024-02-16 01-13-03](https://github.com/nvim-tree/nvim-tree.lua/assets/115842560/570f4887-7863-4b55-a382-baff9cb95e81)

When I do run update it still shows:
![Screenshot from 2024-02-16 01-14-33](https://github.com/nvim-tree/nvim-tree.lua/assets/115842560/2bd89a93-c8b7-419c-9b6f-183ff260b64d)

I am using lazy.nvim for package manager and here is my config:
```lua
return {
  'nvim-tree/nvim-tree.lua',
  version = '*',
  lazy = false,
  config = function()
    require('nvim-tree').setup({
      renderer = { add_trailing = true },
      actions = {
        open_file = {
          quit_on_open = true,
        },
      },
      view = {
        width = 25,
      },
    })
  end,
  vim.keymap.set(
    'n',
    '<leader>xf',
    ':NvimTreeFindFile<cr>',
    { noremap = true, silent = true, desc = 'Find File' }
  ),
  vim.keymap.set(
    'n',
    '<leader>xx',
    ':NvimTreeToggle<cr>',
    { noremap = true, silent = true, desc = 'Toggle Tree' }
  ),
}

```

**Comment by alex-courtis** [comment]
I'm not familiar with lazy, however I am aware that it is very problematic.

I did make an attempt at replication however I'm not sure what I am supposed to be looking for.

Used attached init.lua and ran `:Lazy` `U` several times:
![20240217_135716](https://github.com/nvim-tree/nvim-tree.lua/assets/1505378/bc5dee23-9dcc-47e8-aea3-7da98a5bab5e)

[init.lua.gz](https://github.com/nvim-tree/nvim-tree.lua/files/14317558/init.lua.gz)
